packages/oidc-aws-credentials-provider/src/aws-credentials-provider.ts

```diff
@@ -1,6 +1,6 @@
 import type { AwsCredentialIdentityProvider } from '@smithy/types';
 import type { FromWebTokenInit } from '@aws-sdk/credential-provider-web-identity';
-import { getVercelOidcTokenSync } from './get-vercel-oidc-token';
+import { getVercelOidcTokenSync } from '@vercel/oidc';
 
```

Comment:
I suggest we import `fromWebToken` from `@aws-sdk/credential-provider-web-identity` here rather than a dynamic import. 

It was originally a dynamic import only because `@aws-sdk/credential-provider-web-identity` was an optional peer dependency.